app/static/js/components/Experiments/Experiments.jsx

```diff
@@ -4,6 +4,7 @@ import { withStyles } from "@material-ui/core/styles";
 import IngredientFrequency from "./IngredientFrequency/IngredientFrequencyBarChart";
 import WordCloud from "./WordCloud/WordCloud";
 import styles from "./styles";
+import routes from '../enums/experimentRoutes.js'
```

Comment:
webpack resolve the folder components so you can do components/enums/experimentRoutes if you want. I tend to like that more than using ..

Reply:
Should we add more of these webpack aliases or would it get too wild at some point?

Reply:
this one is probably enough